backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/HealthHandler.kt

```diff
@@ -0,0 +1,28 @@
+package app.ehrenamtskarte.backend.application.webservice
+
+import app.ehrenamtskarte.backend.config.BackendConfiguration
+import app.ehrenamtskarte.backend.projects.database.ProjectEntity
+import app.ehrenamtskarte.backend.projects.database.Projects
+import io.javalin.http.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.jetbrains.exposed.exceptions.ExposedSQLException
+import org.postgresql.util.PSQLException
+import java.lang.System
+import java.net.ConnectException
+
+class HealthHandler(private val config: BackendConfiguration) {
+    fun handle(ctx: Context) {
+        try {
+            transaction {
+                val projectIds = config.projects.map { it.id }
+                val projects = ProjectEntity.find { Projects.project inList projectIds }
+                if (projects.empty()) {
+                    ctx.status(502)
+                }
+            }
+            ctx.status(200)
+        } catch (exception: ExposedSQLException) {
```

Comment:
It could also make sense to change this to `exception: Exception`, to increase the resilience of this endpoint.

Reply:
As you want. Probably any other exception will also return some status != 200.

Reply:
you're right. Otherwise it will return 500 Internal Server Error